stock_putaway_product_form/models/stock_putaway_rule.py

```diff
@@ -0,0 +1,40 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+# Copyright 2020 Sergio Teruel - Tecnativa
+# Copyright 2020 Víctor Martínez - Tecnativa
+
+from odoo import api, fields, models
+
+
+class StockPutawayRule(models.Model):
+    _inherit = "stock.putaway.rule"
+
+    product_tmpl_id = fields.Many2one(
+        comodel_name="product.template",
+        compute="_compute_product_tmpl_id",
+        store=True,
+        inverse=lambda self: self,
```

Comment:
If you add readonly=False this inverse is not needed in v13.0

Reply:
Ok, `readonly=False` added